addons/survey/views/survey_survey_views.xml

```diff
@@ -49,8 +49,12 @@
                     <widget name="web_ribbon" title="Archived" bg_color="bg-danger" attrs="{'invisible': [('active', '=', True)]}"/>
                     <field name="background_image" widget="image" class="oe_avatar"/>
                     <div class="oe_title" style="width: 100%;">
-                        <label for="title"/>
-                        <h1><field name="title" placeholder="e.g. Satisfaction Survey"/></h1>
+                        <label for="title" class="oe_edit_only" string="Survey Title" attrs="{'invisible': [('certification', '=', True)]}"/>
+                        <label for="title" class="oe_edit_only" string="Certification Title" attrs="{'invisible': [('certification', '=', False)]}"/>
+                        <h1>
+                            <field name="title" placeholder="e.g. Satisfaction Survey" attrs="{'invisible': [('certification', '=', True)]}"/>
+                            <field name="title" placeholder="e.g. 'Computer Science Trivia Quiz'" attrs="{'invisible': [('certification', '=', False)]}"/>
```

Comment:
I just tested the check/uncheck of `Is a certification` via the `elearning` module and, in this case, the fields are modified when you check/uncheck the button.

I would completely agree with you in another situation, but we are creating the certification when you go to eLearning > Courses > Certification > Create and we haven't added the button inside of the `survey` module.

Here is a video of the behavior:
https://drive.google.com/file/d/1JGmGEgLk78VAQOpZbEvvlEq1SigqWnBB/view?usp=sharing

For `'`, I completely aggree I'm removing it.

Reply:
But if instead of creating a new certification you edit an existing one it doesn't work (https://drive.google.com/file/d/15dO414M353xs-0y0FN5ldMgdiU7p3xdN/view?usp=sharing )

That's why I would prefer only 1 common placeholder is possible (field in Odoo should be unique in the view or we might have weird behavior)

Reply:
Asked PO for the placeholder, we are keeping the first placeholder.